content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,91 @@
+
+The Access Token have a limited lifetime (about 720 seconds or 2 hours). Once it expires, the client application must use the obtained Client ID and Client Secret to obtain a new Access Token. One indication of the expiration of Access Token, the API will return response Error: '401 Unauthorized HTTP.'  If the user generate new Access Token prior to it expires, then the current Access Token will be invalidated or treated as not Authorized. 
+
+Additionally, user can also change the Client Secret to update the authorization when authorized user lost the Client Secret or the secret is compromised. And lastly, when access to the API must be disabled, user can delete the application association in the API Gateway.
+
+![Access API process](/img/user-guide-for-authorization.png "Process to authenticate and to obtain secure access ")
+
+## Accessing the API Gateway Console
+
+To access the API console, user must login into the [GreenLake Common Cloud](https:\common.cloud.hpe.com), deployed the Data Services Common Cloud Console Application, and onboarded a Storage Array (Alletra, Nimble and Primera) into the organization that is associated with the user account. The user must have the role that is required to perform the intended operation at the instance of DSCC where the storage is deployed. e.g. User must have volume management capability in the Data Ops Management to create a storage volume in US West. 
+
+The **API** **Gateway** menu is available inside **Manage** menu. From **Cloud Console** click on **Menu** to access this **Manage** Menu.
+
+![Access Menu ](/img/accesing-manage-menu-from-console.png "Menu in Cloud Console")
+
+![CCS Menu](/img/accesing-api-gateway-from-manage-menu.png "GreenLake Common Cloud Menu")
+
+The API Gateway provides the following operations:
+
+1. Create and manage API client applications association to obtain:
+
+   1. Instance ID of the DSCC at the particular region (Hexadecimals)
+   2. Client ID (Hexadecimals)
+   3. Client Secret (Hexadecimals)
+   4. URL to the HPE GreenLake Cloud Console end-point (string)
+   5. URL to the HPE DSCC end-point (string)
+2. Generate Access Token, change Client Secret, and delete Access Token.
+
+![API Gateway](/img/api-gateway-block.png "DSCC API Gateway")
+
+### Manage API Client Applications
+
+Each of the API client Applications Connection creates the relationship between the client and server (application). Click on the **Create Credentials** button to generate a client credential to generate the Access Token.
+
+![](/img/create-credentials-button.png)
+
+Inside the Create Credentials menu, click on the V button to show the pull down list and use the mouse to click on the desired application. i.e. For storage select the Data Services Cloud Console. The list also show region where the applications is deployed.
```

Comment:
Change "i.e." to "For example, for storage, select the Data Services Cloud Console."
Change the last sentence to "The list also shows the region where the applications are deployed."
(show**s**, **the** region, **are**)